Plots/Panels/NCL_panel_2.py

```diff
@@ -0,0 +1,123 @@
+"""
+NCL_panel_2.py
+===============
+This script illustrates the following concepts:
+   - Paneling three plots vertically on a page
+   - Adding a common title to paneled plots
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_2_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import cartopy
+import cartopy.crs as ccrs
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+
+
+###############################################################################
+# Import data
+
+# Open netCDF data file using xarray
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc")).isel(time=1)
+
+
+###############################################################################
+# Plot using cartopy and matplotlib
+
+# Generate figure and anxes using Cartopy projection
+# Make three subplots using matplotlib
+projection = ccrs.PlateCarree()
+fig, ax = plt.subplots(3, 1, constrained_layout=True, subplot_kw={"projection": projection})
+
+# Set figure size
+fig.set_size_inches((6, 9.6))
+
+# Define contour levels
+levels = np.linspace(-10, 50, 13)
+
+# Set common plot title
+plt.suptitle("A common title", fontsize=16)
+
+# Display continents
+continents = cartopy.feature.NaturalEarthFeature(name="coastline", category="physical", scale="50m", edgecolor="None",
+                                                 facecolor="lightgray")
+[axes.add_feature(continents) for axes in ax.flat]
+
+# Define the contour levels
+levels = np.arange(-48, 48, 4)
+
+# Using a dictionary makes it easy to reuse the same keyword arguments twice for the contours
+kwargs = dict(
+    levels=levels,  # contour levels specified outside this function
+    xticks=np.arange(-180, 181, 30),  # nice x ticks
+    yticks=np.arange(-90, 91, 30),  # nice y ticks
+    transform=projection,  # ds projection
+    add_colorbar=False,  # don't add individual colorbars for each plot call
+    add_labels=False,  # turn off xarray's automatic Lat, lon labels
+    colors="k",  # note plurals in this and following kwargs
+    linestyles="-",
+    linewidths=0.5,
+)
```

Comment:
I would put this parenthesis on line 69 and remove the additional comma after 0.5 for formatting consistency

Reply:
Done!